core-blocks/latest-comments/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import './style.scss';
+import edit from './edit';
+
+export const name = 'core/latest-comments';
+
+export const settings = {
+	title: __( 'Latest Comments' ),
+
+	description: __( 'Shows a list of your site\'s most recent comments.' ),
+
+	icon: 'list-view',
+
+	category: 'widgets',
+
+	keywords: [ __( 'recent comments' ) ],
+
+	supports: {
+		html: false,
+	},
+
+	defaultAttributes: {
```

Comment:
`defaultAttributes` is not supported. It should be removed as it takes no effect.